pfinance/functions.py

```diff
@@ -63,6 +63,38 @@ def future_value_series(
     return payment * start_modifier * (((1 + effective_rate) ** total_periods) - 1) / effective_rate
 
 
+def present_value(
+    payment: float,
+    interest_rate: float,
+    periods: int,
+    future_value: float = 0,
+    start_of_period: bool = False
```

Comment:
nit: I prefer a comma at the end of multi line parameter lists, since it means if a new parameter is added or removed it will be a single line diff